src/main/java/org/fundacionjala/coding/franz/SpinWords.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.franz;
+
+/**
+ * This is a class for Spin words.
+ */
+public class SpinWords {
+    private static final int LIMIT_TAM = 4;
+
+    /**
+     * this method is a spin words.
+     *
+     * @param sentence is the words that use for excercise
+     * @return words reverse if is a length major to 5 for word
+     */
+    public String spinWords(String sentence) {
+
+        String[] words = sentence.split(" ");
+        StringBuilder wordComplete = new StringBuilder();
+
+        if (words.length != 1) {
+            wordComplete.append(reverseWord(words[0])).append(" ");
+        } else {
+            wordComplete.append(reverseWord(words[0]));
+        }
+        for (int i = 1; i < words.length; i++) {
+            wordComplete.append(reverseWord(words[i])).append(" ").toString();
+        }
+        return wordComplete.toString().trim();
+    }
+
+    /**
+     * this method reverse a word.
+     *
+     * @param word is a word that need reverse
+     * @return the word reverse
+     */
+    public String reverseWord(String word) {
+        String wordRev = word.length() > LIMIT_TAM ? new StringBuilder(word).reverse().toString()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "wordRev". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)